commcare_connect/reports/views.py

```diff
@@ -401,22 +401,40 @@ def dashboard_stats_api(request):
     filterset = DashboardFilters(request.GET)
 
     # Use the filtered queryset to calculate stats
-    queryset = UserVisit.objects.all()
+    visit_queryset = UserVisit.objects.all()
+    payment_queryset = Payment.objects.all()
```

Comment:
I might have missed this in a previous PR, but i think both of these need to filter for opportunities where `is_test` is `False`, and I think the payment one needs to filter for `opportunity_access` not being null (since that represents a payment to a user rather than an org)

Reply:
thanks, updated.

In the future we'll also want to add an indicator for org payments. is it clear how to filter out test opportunities there? Would it be through the invoice field?

Reply:
Hm, I don't know that we have a good way to filter for that. I think we try to be clever about orgs with real opportunities, but not sure how efficient that query is (probably not horrible but its at least one extra join to get the opportunity for an org). We might want to eventually store that at the org level, although I can imagine eventually orgs will have some test payments and some real ones.

Reply:
It looks like you figured this out, but yeah. That sounds right.